components/logger.rst

```diff
@@ -52,23 +52,99 @@ Advanced settings:
 Hardware UARTs
 --------------
 
-The logger component makes use of platform-specific hardware UARTs for serial logging.
-By default, the logger will occupy ``UART0``. The ESP32 has three hardware UARTs, all of
-which can be used for both transmit and receive. The ESP8266 only has two hardware UARTs,
-one of which is transmit-only. The ESP8266 ``UART0`` can also be 'swapped' to TX/RX on the
-CTS/RTS pins, if you need to use GPIO1 and GPIO3 for something else. Note that the common
-NodeMCU boards have their USB-UART Adapters fixed to the default GPIOs used by ``UART0``,
-so if you use anything else you will not get log messages over the on-board USB.
-
-Possible Hardware UART configurations:
-
-- ``UART0`` - TX: GPIO1, RX: GPIO3
-- ``UART0_SWAP`` - TX: GPIO15, RX: GPIO13  (Only on ESP8266)
-- ``UART1`` - TX: GPIO2, RX: None  (Only on ESP8266)
-- ``UART1`` - TX: GPIO9, RX: GPIO10  (Only on ESP32)
-- ``UART2`` - TX: GPIO16, RX: GPIO17  (Only on ESP32 but not ESP32S2, ESP32S3 or ESP32C3)
-- ``USB_CDC`` - uses the USB CDC driver (Only on ESP32S2 and ESP32S3)
-- ``USB_SERIAL_JTAG`` - uses the USB Serial/JTAG driver (Only on ESP32S3 and ESP32C3)
+The logger component makes use of platform-specific hardware UARTs for serial logging. For example, the ESP32
+has three hardware UARTs, all of which can be used for both transmit and receive. The ESP8266 only has two
+hardware UARTs, one of which is transmit-only. The ESP8266's ``UART0`` can also be "swapped" to TX/RX on the
+CTS/RTS pins in the event that you need to use GPIO1 and GPIO3 for something else.
+
+Note that many common boards have their USB-to-serial adapters fixed to the default GPIOs used by ``UART0``,
+so if you use any other configuration you will not get log messages over the on-board USB.
+
+Default UART GPIO Pins
+**********************
+
+.. list-table::
```

Comment:
I think swap the rows and columns here to make it a vertical table instead of quite a wide horizontal one.

Reply:
SGTM 😄

Reply:
Done